dokka-subprojects/analysis-java-psi/src/main/kotlin/org/jetbrains/dokka/analysis/java/parsers/JavaDocCommentParser.kt

```diff
@@ -73,18 +74,38 @@ internal class JavaPsiDocCommentParser(
         analysedElement: PsiNamedElement
     ): TagWrapper? {
         val paramName = tag.dataElements.firstOrNull()?.text.orEmpty()
-
-        // can be a PsiClass if @param is referencing class generics, like here:
-        // https://github.com/biojava/biojava/blob/2417c230be36e4ba73c62bb3631b60f876265623/biojava-core/src/main/java/org/biojava/nbio/core/alignment/SimpleProfilePair.java#L43
-        // not supported at the moment
```

Comment:
Shoot, I think it's one of my comments that I didn't turn into a TODO/issue, so I forgot to fix it before the merge :sweat_smile: